app/src/main/java/com/onelist/MainViewModel.kt

```diff
@@ -51,8 +51,8 @@ class MainViewModel(var itemService: IItemService = ItemService()) : ViewModel()
         }
         item.itemID = document.id
         val handle = document.set(item)
-        handle.addOnSuccessListener { Log.d("Firebase", "Item Saved") }
-        handle.addOnFailureListener { Log.d("Firebase", "Item save failed $it") }
+        handle.addOnSuccessListener { Log.i("Firebase", "Item Saved") }
+        handle.addOnFailureListener { Log.e("Firebase", "Item save failed $it") }
```

Comment:
Great.  Logging at the correct level is important!  Failures should be less common, and easier to find in logs, than successes.